### PR TITLE
Db 1242 exclude downward adjustments

### PR DESCRIPTION
**Pull request by alburde1**
- [x] Code Review Complete

This PR removes unneeded USSGL checks from the relevant rules so they match the validation table.

### CODE REVIEW

**Comment by klrBAH** [comment]
Can one of the admins verify this patch?

**Comment by jworcestBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

File: tests/integration/baseTestValidator.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def setUpClass(cls):
         """Set up resources to be shared within a test class"""
         #TODO: refactor into a pytest class fixtures and inject as necessary
         # Prevent interface being reused from last suite
-        BaseInterface.interfaces = None
+        # BaseInterface.interfaces = None
```
Comment (jworcestBAH) [inline tests/integration/baseTestValidator.py]: We should probably remove this line rather than just commenting it.

Comment (alburde1) [inline tests/integration/baseTestValidator.py]: That sounds like a good idea. It's done in the latest commit.

**Comment by jworcestBAH** [comment]
Looks good, merging

**Pull request merged**